afkak/client.py

```diff
@@ -18,15 +18,13 @@
 
 High level network client for an Apache Kafka Cluster.
 """
-from __future__ import absolute_import, print_function
 
```

Comment:
Python 2 reference in comment at [line 912](https://github.com/ciena/afkak/blob/0dc1a5904492c58d3a4b0fc113fde2638a309f31/afkak/client.py#L912).